contracts/api-spec.yaml

```
@@ -268,7 +268,7 @@ paths:
         - $ref: '#/components/parameters/EncoderSerialNumber'
       responses:
         '200':
-          description: Successful retrieval of the encoder
+          description: Successful creation the encoder
```

I swore you changed this, but I have no way to prove it. I think your force push reversed the change
```suggestion
          description: Successful retrieval of the encoder
```

---

I think merging is playing tricks on me